src/components/Layout/footer.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import styles from './footer.module.css';
+import Typography from '@mui/material/Typography';
 
 const Footer = () => {
 	return (
 		<div className={styles.footerContainer}>
-			<div>This footer is always at the bottom of the content or page and links can go here</div>
+			<Typography component='h1'>Group allocation system</Typography>
```

Comment:
tbh do we even need to have the footer any more, I originally added it but don't know if it's still necessary, maybe we should just comment it out in the layout component?